packages/frontend/src/components/dialogs/ProxyConfiguration/styles.module.scss

```diff
@@ -0,0 +1,74 @@
+.proxyDialogBody {
+  display: flex;
+}
+
+.proxyList {
+  flex-grow: 1;
+  overflow-y: scroll;
```

Comment:
You can't see saved proxies on not-so-tall screens

![image](https://github.com/user-attachments/assets/11d6a09d-d629-4afb-a7a4-aabe3d30f855)

I suggest to make the entire dialog body scrollable instead. And perhaps also move the "add proxy" section to the dialog body.

Reply:
If the whole body is scrollable it might result in a hidden add button or a hidden enabe/disable button if you have multiple proxies. That seems more problematic to me than the edge case of screen height < 400px

Reply:
Can one call it "hidden" though? You can scroll to it. And if you already have proxies, you already know that there is an "add" button.
The way it is right now, it's just not possible to see the proxies on short window size, no matter how you scroll.

Reply:
Maybe we add a media query for that edge case?

Reply:
Added one for screen height <= 500px